src/styles/main.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import "blocks/header";
+@import "blocks/card";
+@import "blocks/products-main";
+@import "blocks/stars";
+@import "blocks/reset";
+@import "blocks/nav";
```

Comment:
Consider using the right order for imports: 
1) For variables;
2) For basic styling, entire page styling
3) Blocks one by one from top to bottom